packages/modals/src/views/Close.js

```diff
@@ -13,6 +13,8 @@ import ModalStyles from '@zendeskgarden/css-modals';
 import { composeEventHandlers } from '@zendeskgarden/react-selection';
```

Comment:
yep good catch

Reply:
so ModalContainer uses react-selection and I feel it'll be wasted effort to move ModalContainer over considering this will be deprecated shortly so might leave this in for now